### ISSUE TITLE
Build function not working

### ISSUE THREAD

**Issue by iGEM-Aachen-User** [BristolIGEM2017/IDE#3]
When trying to run the built script I get the following error.
![image](https://user-images.githubusercontent.com/33207667/43477455-85e83ee2-94fb-11e8-8d72-9f17f9a70637.png)

**Comment by AlbertWigmore** [comment]
What python version are you running?
The recursive argument in glob is not available in python 2.7 or Python 3.4< so this may be the issue.

To find out your python version run
```
python --version
```

**Comment by iGEM-Aachen-User** [comment]
Yes I found the error now, I was running 3.3 or something. Thanks!

But now I got the problem that:
1. it is not creating the template and js folder with its coresponding files. (that is not a big problem as I can just manually edit these)
2. when I upload the files onto the igem website I get this http://2018.igem.org/Team:Aachen
